src/App.js

```diff
@@ -1,54 +1,83 @@
 import { useEffect, useState } from "react";
-import './App.css';
-import logo from './mlh-prep.png'
+import "./App.css";
+import logo from "./mlh-prep.png";
+import WeatherMap from "./components/WeatherMap";
 
 function App() {
   const [error, setError] = useState(null);
   const [isLoaded, setIsLoaded] = useState(false);
-  const [city, setCity] = useState("New York City")
+  const [city, setCity] = useState("New York City");
```

Comment:
Revert the change on this line please. We'll do a single PR for all styling fixes once #17 is merged in.